### PR TITLE
feat: 520-beshared-add-getuserbyreferalcode-query-api

### PR DESCRIPTION
**Pull request by daisai3**
- task: https://trello.com/c/yvvczEee/520-beshared-add-getuserbyreferalcode-query-api

### CODE REVIEW

**Review by aaron-alderman — CHANGES_REQUESTED** [review]
Change to GetUserByReferralCodes and return array of items

**Comment by daisai3** [comment]
> Change to GetUserByReferralCodes and return array of items

It was renamed to GetUserByReferralCodes.

File: graphql/getUserByReferralCodes.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { gql } from "@apollo/client/core";
+
+const GET_USER_BY_REFERRAL_CODES = gql(`
```
Comment (aaron-alderman) [inline graphql/getUserByReferralCodes.ts:3]: Will need plural to keep consistent
`GET_USERS_BY_REFERRAL_CODES`
`GetUsersByReferralCodes`
`getUsersByReferralCodes`

**Review by aaron-alderman — CHANGES_REQUESTED** [review]

File: graphql/getUserByReferralCodes.ts (hunk 1 above)
Comment (daisai3) [reply graphql/getUserByReferralCodes.ts]: Fixed it

**Review by leckylao — APPROVED** [review]

**Review by aaron-alderman — APPROVED** [review]

**Pull request merged**